test_list_comprehension_exercises.py

```diff
@@ -0,0 +1,75 @@
+                                                                     233, 234, 235, 236, 237, 238, 239, 243, 253, 263,
+                                                                     273, 283, 293, 300, 301, 302, 303, 304, 305, 306,
+                                                                     307, 308, 309, 310, 311, 312, 313, 314, 315, 316,
+                                                                     317, 318, 319, 320, 321, 322, 323, 324, 325, 326,
+                                                                     327, 328, 329, 330, 331, 332, 333, 334, 335, 336,
+                                                                     337, 338, 339, 340, 341, 342, 343, 344, 345, 346,
+                                                                     347, 348, 349, 350, 351, 352, 353, 354, 355, 356,
+                                                                     357, 358, 359, 360, 361, 362, 363, 364, 365, 366,
+                                                                     367, 368, 369, 370, 371, 372, 373, 374, 375, 376,
+                                                                     377, 378, 379, 380, 381, 382, 383, 384, 385, 386,
+                                                                     387, 388, 389, 390, 391, 392, 393, 394, 395, 396,
+                                                                     397, 398, 399, 403, 413, 423, 430, 431, 432, 433,
+                                                                     434, 435, 436, 437, 438, 439, 443, 453, 463, 473,
+                                                                     483, 493, 503, 513, 523, 530, 531, 532, 533, 534,
+                                                                     535, 536, 537, 538, 539, 543, 553, 563, 573, 583,
+                                                                     593, 603, 613, 623, 630, 631, 632, 633, 634, 635,
+                                                                     636, 637, 638, 639, 643, 653, 663, 673, 683, 693,
+                                                                     703, 713, 723, 730, 731, 732, 733, 734, 735, 736,
+                                                                     737, 738, 739, 743, 753, 763, 773, 783, 793, 803,
+                                                                     813, 823, 830, 831, 832, 833, 834, 835, 836, 837,
+                                                                     838, 839, 843, 853, 863, 873, 883, 893, 903, 913,
+                                                                     923, 930, 931, 932, 933, 934, 935, 936, 937, 938,
+                                                                     939, 943, 953, 963, 973, 983, 993]
+
+
+def test_count_the_number_of_spaces_in_string_no_space_in_string():
+    assert count_the_number_of_spaces_in_string("BARTEK") == 0
+
+
+def test_count_the_number_of_spaces_in_string_one_space_in_string():
```

Comment:
It's always good to write a test for edge cases, like in this example empty string "".

Reply:
Test added and it is works